src/OrbitGl/App.h

```diff
@@ -645,6 +645,11 @@ class OrbitApp final : public DataViewFactory,
   absl::flat_hash_map<std::pair<std::string, std::string>, orbit_base::Future<ErrorMessageOr<void>>>
       symbols_currently_loading_;
 
+  // Set of modules where an symbol loading error has occurred. The module identifier is file path
```

Comment:
Set of modules where **a** symbol ...

Reply:
The module identifier **consists** in a file path ...

Reply:
done